config/index.js

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash';
+
+import variantBase from './variants/_base';
+
+export const VARIANTS = [
+  'development',
+  'demo',
+  'production',
```

Comment:
They're the same concept.  `NODE_ENV` makes zero sense to me for a non-Node env...